pyiron_base/jobs/job/runfunction.py

```diff
@@ -454,8 +454,10 @@ def run_job_with_runmode_srun(job):
 def run_job_with_runmode_flux(job, executor, gpus_per_slot=None):
     if not flux_available:
         raise ModuleNotFoundError(
-            "No module named 'flux'. No linux you can install flux via conda."
-            + "'conda install -c conda-forge flux'"
+            "No module named 'flux'. Running in flux mode is only available on Linux;"
+            "For CPU jobs, please use `conda install -c conda-forge flux-core`; for "
+            "GPU support you will additionally need "
+            "`conda install -c conda-forge flux-sched`"
```

Comment:
```suggestion
            "`conda install -c conda-forge flux-sched libhwloc=*=cuda*`"
```

Reply:
`libhwloc` is required to be build with CUDA support so flux recognizes the GPUs.